src/components/accordion/_accordion.scss

```diff
@@ -283,6 +273,44 @@
   .#{$prefix}--accordion__title.#{$prefix}--skeleton__text {
     margin-bottom: 0;
   }
+
+  @include accordion-colors--x;
+}
+
+@mixin accordion-colors--x {
```

Comment:
When we release `v10`, the default values of feature flags will be flipped. It means that `v10` styles will be effective by default.

Reply:
That's specifically why I asked this... Carbon users use these mixins directly so when the feature flag is turned off we'd have to go and update all reference to these mixins... seems very tedious from a user perspective.

Reply:
Might be worth submitting a request to: https://github.com/carbon-design-system/carbon-upgrade and we can try and include it 👍

For reference, [here](https://github.com/carbon-design-system/carbon-upgrade/blob/master/src/migrations/carbon-colors/0.0.1-alpha.32.js#L21-L31) is how we used it to update color names. We also used it for updating namespaces [here](https://github.com/carbon-design-system/carbon-upgrade/blob/master/src/migrations/carbon-layout/0.0.1-alpha.32.js#L24) for layout.

In v10, we will also be more strict around our sassdoc annotations so that it is clear what is intended for public or private access. These component-level mixins aren't intended for public access, as far as I'm aware. @asudoh feel free to correct me if I'm wrong here 👍

Reply:
^ Right, these are just for conditional import.

Reply:
The color mixins are 100% meant to be public, that's what my request is for. I want the ability to call a mixin that would change only the colors of a component. If they are private then they provide no value to WH which is why having the `--x` and changing it with the feature flags does seem realistic for manageability.

@joshblack isn't my request on this repo the request for the update? I don't understand why you're asking to have a duplicate thread.

Reply:
@mattrosno can you chime in here... this is a very big concern for WH and would basically make a lot of this work moot for us. Were you also under the impression that these would be private mixins?